src/api/services/firstAPI.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { TableProps } from '../../features/teams/TeamsPage';
+const BASE_URL_TEAMS = process.env.REACT_APP_BASE_URL;
```

Comment:
Name `BASE_URL` is more appropriate, but` process.env.REACT_APP_BASE_URL` is a variable that is already common.

You can improve this, in some steps:
- [ ] Add `BASE_URL_LIMIT=10` in `.env`
- [ ] Create `libs/axios.ts` & set default `baseURL`, like:
```javascript
import $axios from 'axios';

const axios = $axios.create();
axios.defaults.baseURL = process.env.REACT_APP_BASE_URL;

export default axios;
```
- [ ] Create `utils/filters.ts` with default params, like:
```javascript
import { Properties } from 'types';

export const defaultFilters: Properties = {
  page: 1,
  limit: +process.env.REACT_APP_BASE_URL_LIMIT || 10,
};
```
- [ ] Create `hooks/filters.ts`, [like](https://github.com/streamich/react-use/blob/master/src/useSetState.ts):
```javascript
import { useCallback, useState } from 'react';

export default <T extends object>(
    initialState: T = {} as T
): [T, (patch: Partial<T> | ((prevState: T) => Partial<T>)) => void] => {
    const [state, set] = useState<T>(initialState);
    const setState = useCallback((patch) => {
        set((prevState) =>
            Object.assign({}, prevState, patch instanceof Function ? patch(prevState) : patch)
        );
    }, []);

    return [state, setState];
};;
```
```javascript
import { useFilters } from 'hooks';
import { defaultFilters } from 'utils';

const [filters, setFilters] = useFilters({ ...defaultFilters })
```
- [ ] Use `query-string` package to get url with an object into a query string
```javascript
import { default as queryString } from 'query-string';

getList: async (query: Properties) => {
    const { data } = await axios.get(
        queryString.stringifyUrl(
            {
                url: `/teams,
                query,
            },
            { skipEmptyString: true, skipNull: true },
        )
    );
    
    return data;
},
```

### Some useful libraries to solve your issues
- [react-use](https://www.npmjs.com/package/react-use)
- [query-string](https://www.npmjs.com/package/query-string)